x-pack/plugins/session_view/public/components/process_tree_alerts_filter/index.tsx

```diff
@@ -133,14 +146,20 @@ export const ProcessTreeAlertsFilter = ({
       <EuiFlexGroup alignItems="center">
         <EuiFlexItem css={filterStatus} style={{ paddingLeft: '16px' }}>
           <EuiText size="s">
-            Showing{' '}
-            <span
-              className={'text'}
-              data-test-subj="sessionView:sessionViewAlertDetailsFilterStatus"
-            >
-              {alertCountsStatusText}
-            </span>{' '}
-            alerts
+            <FormattedMessage
+              id="sessionView.alertDetailsCountStatus"
+              defaultMessage="Showing {alertCountsStatusElem} alerts"
+              values={{
+                alertCountsStatusElem: (
```

Comment:
I found a way how to make the translations more dynamic according to this style guide ->https://github.com/elastic/kibana/blob/main/packages/kbn-i18n/GUIDELINE.md